docs/home/contribute/localization.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Localizing Kubernetes Documentation
+approvers:
+- chenopis
+- zacharysarah
+---
+
+We're happy to add localizations (l10n) of Kubernetes documentation to the website!
+
+Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+
+## Workflow  
+
+All l10n work must be stored and tracked within the [Kubernetes organization](https://github.com/kubernetes).
+
+### Basis for localizations
+
+Localizations must source from the English files for the [most recent major release](https://kubernetes.io/docs/home/supported-doc-versions/#current-version).
+
+**Note:** To find the most recent release's documentation source files:
+1. Navigate to https://github.com/kubernetes/website.
+2. Select the `release-1.X` branch for the most recent version.
+
+    For example, the branch for Kubernetes v1.9 docs is `release-1.9`.
+
+Source files reside in the `/docs/` directory.
+{: .note}
+
+### Repository
+
+A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs/cn](https://github.com/kubernetes/kubernetes-docs-cn).
+
+**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
+{: .note}
+
+### Project
+
+Teams must track their overall progress with a [GitHub project](https://help.github.com/articles/creating-a-project-board/).
+
+Projects must include columns for:
+- To do
+- In progress
+- Done
+
+For example: the [Chinese localization project](For example: https://github.com/kubernetes/kubernetes-docs-cn/projects/1).
```

Comment:
bad format :) `For example: ` in the parentheses is redundant.

Reply:
Good catch!